### DISCUSSION TITLE
Tawk.to Application

### DISCUSSION THREAD

**Discussion by kaganergovan** [elrumo/macOS_Big_Sur_icons_replacements#1300] [Icon request] [answered]
Hi dear community!

I'm searching for tawk.to App's icon but I didn't find it. Anyone can help me please?

Sincerely,

**Comment by IceGoddd** [comment, marked as answer]
<img width="512" alt="tawkto" src="https://user-images.githubusercontent.com/70185179/137601321-d02e0cc6-6201-4a52-ad23-d89d9e836808.png">
<img width="512" alt="tawkto2" src="https://user-images.githubusercontent.com/70185179/137601324-3dc65680-a4a7-4033-afff-6f4bd4fb6784.png">

**Reply by kaganergovan** [reply]
You are beesstt, thanks so much @IceGoddd!
